<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlueprintCalculatorWidget</class>
 <widget class="QWidget" name="BlueprintCalculatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blueprint Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="BlueprintPixmapLabel" name="blueprintPixmapLabel">
       <property name="text">
        <string>Drag a blueprint or a product here.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="blueprintNameLabel">
           <property name="text">
            <string>&lt;h2&gt;No Blueprint Selected&lt;/h2&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="InfoButton" name="blueprintInfoButton" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>ME:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="meSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-2147483647</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>PE:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="peSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-2147483647</number>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="manufacturingTab">
      <attribute name="title">
       <string>Manufacturing</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
       <item>
        <layout class="QVBoxLayout" name="manufacturingVerticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="productHorizontalLayout">
           <item>
            <widget class="QLabel" name="productPixmap">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="productGridLayout">
             <item row="0" column="1">
              <widget class="InfoButton" name="productInfoButton" native="true"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="productNameLabel">
               <property name="text">
                <string>&lt;h2&gt;No Blueprint Selected&lt;/h2&gt;</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="productPortionSizeLabel">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Configuration</string>
           </property>
           <layout class="QFormLayout" name="formLayout_5">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Runs:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="manufacturingRunsSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Total</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Materials Cost:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Sell Price:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Gross Profit:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Manufacturing Time:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="manufacturingTimeLabel">
              <property name="text">
               <string>0 min</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="materialsCostLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">N/A ISK</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="sellPriceLabel">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">N/A ISK</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="profitLabel">
              <property name="text">
               <string notr="true">N/A ISK</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Basic Materials</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTreeWidget" name="basicMaterialsTable">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Basic Materials Cost:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="basicMaterialsCostLabel">
                <property name="text">
                 <string notr="true">N/A ISK</string>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Extra Materials</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTreeWidget" name="extraMaterialsTable">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_4">
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Extra Materials Cost:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="extraMaterialsCostLabel">
                <property name="text">
                 <string notr="true">N/A ISK</string>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InfoButton</class>
   <extends>QWidget</extends>
   <header>infobutton.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlueprintPixmapLabel</class>
   <extends>QLabel</extends>
   <header>blueprintpixmaplabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
